lib/monitoring/checkers.go

```diff
@@ -72,6 +73,10 @@ type Config struct {
 	LowWatermark uint
 	// HighWatermark is the disk usage critical limit percentage of monitored directories
 	HighWatermark uint
+	// ServiceUID is the UID of the service (planet) user.
+	ServiceUID string
```

Comment:
nit: it would be more practical to store these are final values that need no further processing (i.e. parsing) and do the validation as early as possible - e.g. when accepting values on command line. As far as I can tell these are always integers.